frontend/src/lib/components/modals/first-time-setup/FirstTimeSetupModal.svelte

```diff
@@ -0,0 +1,115 @@
+<script lang="ts">
+  import { mdiTimerSandFull, mdiWeb } from '@mdi/js';
+  import { SlideToggle } from '@skeletonlabs/skeleton';
+  import { T } from '@tolgee/svelte';
+
+  import LanguageSelector from './LanguageSelector.svelte';
+
+  import SvgIcon from '$lib/components/SVGIcon.svelte';
+  import { queueAutoStart } from '$lib/store/settingsStore';
+  import { SetNewUserSetupComplete } from '$wailsjs/go/settings/settings';
+  import { BrowserOpenURL } from '$wailsjs/runtime/runtime';
+
+  export let parent: { onClose: () => void };
+
+  const OpenWelcomeGuide = () => {
+    BrowserOpenURL('https://docs.ficsit.app/satisfactory-modding/latest/ForUsers/Welcome.html');
+  };
+
+  function onClose() {
+    SetNewUserSetupComplete(true);
+    parent.onClose();
+  }
+</script>
+
+<div
+  style="max-height: calc(100vh - 3rem); max-width: calc(100vw - 3rem);"
+  class="w-[48rem] card flex flex-col gap-6"
+>
+  <header class="card-header font-bold text-2xl text-center">
+    <T defaultValue="Welcome to the Satisfactory Mod Manager!" keyName="first_time_setup.title" />
+  </header>
+  <section class="px-4">
+    <p class="text-base text-center">
+      <T
+        defaultValue="Select your preferences to get started."
+        keyName="first_time_setup.intro"
+      />
+    </p>
+  </section>
+  <section class="px-4 overflow-y-auto">
+    <ul class="list">
+      <li>
+        <span class="badge bg-tertiary-500">
+          <SvgIcon
+            class="h-5 w-5"
+            icon={mdiTimerSandFull}
+          />
+        </span>
+        <span class="flex-auto">
+          <p class="text-lg"><T defaultValue="When I add or remove a mod, or switch profiles..." keyName="first_time_setup.option.queue-auto-start.title" /></p>
+          <p class="text-base">
```

Comment:
Doesn't need to be wrapped in `<p>` (which is also unclosed) and `<span>`, you can set the classes on SlideToggle directly. `text-base` also doesn't have any effect because that's already the current font size.

Reply:
I am surprised editor warnings didn't tell me it was unclosed, it does for a lot of stuff